rythmbadge/request_handler_lib.c

```diff
@@ -225,6 +225,7 @@ static ret_code_t receive_notification_fifo_peek(receive_notification_t* receive
 	}
 	
 	uint8_t tmp[notification_size];
+	memset(tmp, 0, notification_size * sizeof(uint8_t));
```

Comment:
just a nitpick, no need to do the calculation, `sizeof(tmp)` will yield the same result as `notification_size * sizeof(uint8_t)`. Feel free to ignore as this will be const evaluated anyways